pom.xml

```diff
@@ -250,6 +250,19 @@
 					</systemProperties>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-failsafe-plugin</artifactId>
+				<version>2.22.2</version>
```

Comment:
From what I recall, the latest failsafe-plugin version is somwhere in 3.0.0 range. Any reason why you needed to use an older version of this dependency?

Reply:
two reasons of small importance:
 * to be consistent with existing usage of maven-surefire-plugin 2.22.2
 * because latest release of maven-failsafe-plugin is 3.0.0-M7 and not a "proper" release

https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-failsafe-plugin